web/src/main/java/se/inera/intyg/webcert/web/web/controller/facade/ConfigController.java

```diff
@@ -78,7 +81,7 @@ public Response getConfiguration() {
             .filter((banner -> banner.getApplication() == Application.WEBCERT))
             .collect(Collectors.toList());
 
-        return Response.ok(new ConfigurationDTO(version, banners, sakerhetstjanstIdpUrl, cgiFunktionstjansterIdpUrl)).build();
+        return Response.ok(new ConfigurationDTO(version, banners, ppHost, sakerhetstjanstIdpUrl, cgiFunktionstjansterIdpUrl)).build();
```

Comment:
Could you add a test to verify that the ppHost is added to the response correctly? A simple test that can will be a useful regression test to make sure no one mixes up the strings...